website/content/docs/commands/agent.mdx

```diff
@@ -65,6 +65,12 @@ via CLI arguments. The `agent` command accepts the following arguments:
 - `-consul-client-http-check-name=<name>`: Equivalent to the
   [client_http_check_name] config option.
 
+- `-consul-client-failures-before-critical=<num>`: Equivalent to the
+  [client_failures_before_critical] config option.
+
+- `-consul-client-failures-before-warning=<num>`: Equivalent to the
+  [client_failures_before_warning] config option.
```

Comment:
Having the little empty `[]` lets a lot of commonly used tools pick this up as needing syntax highlighting:

```suggestion
- `-consul-client-failures-before-critical=<num>`: Equivalent to the
  [client_failures_before_critical][] config option.

- `-consul-client-failures-before-warning=<num>`: Equivalent to the
  [client_failures_before_warning][] config option.
```

Reply:
Is this correct? I see all the other links without this?

Reply:
Oh sorry maybe that was confusing. It doesn't impact the links on the actual website at all!

But depending on what editor / LSP / etc you use, having this trailing `[]` can make the difference between the `[client_failures_before_warning]` being shown as plain text vs highlighted as a link when you're editing the markdown. (Emacs + `markdown-mode` is one example, I know personally, but @angrycub has pointed out other cases I can't recall.)